src/content/docs/en/reference/cli-reference.mdx

```diff
@@ -196,12 +196,32 @@ If you started your project using [the `create astro` wizard](/en/install/auto/#
 
 Runs Astro's development server. This is a local HTTP server that doesn't bundle assets. It uses Hot Module Replacement (HMR) to update your browser as you save changes in your editor.
 
+<h3>Flags</h3>
+
+Use these flags to customize the behavior of the command.
+
+#### `--viteMode <mode>`
+
+It defines [mode](https://vitejs.dev/config/shared-options.html#mode) property of the internal Vite config.
+
+<ReadMore>Read more about [Vite modes](https://vitejs.dev/guide/env-and-mode#modes).</ReadMore>
```

Comment:
```suggestion
#### `--viteMode <mode>`
<p><Since v="4.8.0" /></p>

Configures the [`mode`](https://vitejs.dev/config/shared-options.html#mode) property of the internal Vite config. Defaults to `development`.

<ReadMore>Read more about [Vite modes](https://vitejs.dev/guide/env-and-mode#modes).</ReadMore>
```
We would normally start flag descriptions with "Configures" or "Sets". I think either word is fine here! You could even use "Overrides" if you want to emphasize that aspect of it, and if it will only ever be used to override the default. ("Define" is more like a definition of something, which feels more permanent and not changeable.)